test/cookbooks/scponly-test/recipes/scponly_chroot.rb

```diff
@@ -1,9 +1,15 @@
+include_recipe 'osl-selinux'
+
 scponly_user 'scponly_test_chroot' do
   write_dir 'write'
   public_key node['scponly']['public_key']
   chroot true
 end
 
+selinux_fcontext '/var/lib/chroots/home/scponly_test_chroot/.ssh/id_rsa-scponly_user-scponly_test_chroot' do
+  secontext      'httpd_sys_rw_content_t'
```

Comment:
I'm not sure if the secontext I set on the private key is what should be used. However, as it currently stands it passes the tests where before it was failing.

Reply:
I believe `ssh_home_t` is better. Can you please change that?

Reply:
Yes just made the changes to the context.